### PR TITLE
feature(#295): Add KtFieldRadioGroup Scoped Slot Support

### LINKED ISSUE #295: KtFieldRadioGroup: Support Scoped Slot
Necessary for:

* `yoda/src/user-panel-frontend/src/features/checkout/components/CheckoutAddressSelection.vue`
* `yoda/src/user-panel-frontend/src/features/checkout/components/ShippingMethodForm.vue`
* `yoda/src/user-panel-frontend/src/features/checkout/components/payment/PaymentMethodSelection.vue`

---

See #273

### PR DESCRIPTION
**Pull request by HansAuger**
Adds scoped slots to `KtFieldRadioGroup` and `KtFieldToggleGroup`

Not sure how to document the `option` binding in a good way though

Closes #295

### CODE REVIEW

File: packages/documentation/pages/components/form-fields.vue [hunk 1]
```diff
@@ -279,6 +285,7 @@ const components: Array<{
 	formKey: string
 	name: string
 	supports: Kotti.Field.Supports
+	slots?: Array<string>
```
Comment (carsoli) [inline packages/documentation/pages/components/form-fields.vue:288]: a bit confused here 

is it not the slots object that has name ad content here

Comment (carsoli) [reply packages/documentation/pages/components/form-fields.vue:288]: ah no, it's an array of strings

File: packages/documentation/pages/components/form-fields.vue [hunk 2]
```diff
@@ -439,7 +449,16 @@ const generateCode = (component: ComponentValue) =>
...
-		'/>',
+		...(component.slots.length === 0
+			? ['/>']
+			: [
+					'>',
+					...component.slots.map(
+						({ name, content }) =>
+							`\t<template v-slot:${name} v-text="${content}" />`,
+					),
+					`</ ${component.name}>`,
```
Comment (carsoli) [inline packages/documentation/pages/components/form-fields.vue:460]: heh? `</ `??

**Review by RueRivoli — APPROVED** [review]

**Review by carsoli — APPROVED** [review]

**Comment by carsoli** [comment]
reviewed and tested thoroughly

merging

**Pull request merged**